core/execution/market.go

```diff
@@ -1911,15 +1911,22 @@ func (m *Market) resolveClosedOutParties(ctx context.Context, distressedMarginEv
 		// which have acceptable margins
 		okPos, closed = m.risk.ExpectMargins(distressedMarginEvts, m.lastTradedPrice.Clone())
 
+		parties := make([]string, 0, len(okPos))
 		if m.log.GetLevel() == logging.DebugLevel {
```

Comment:
we only send the event if debug logging is on?  can't be right can it?

Reply:
The send bit is outside of the `if`

Reply:
right but the parties was only added to if debug logging was on, but I think you've fixed this now?